sources/packages/api/src/route-controllers/institution/institution.controller.ts

```diff
@@ -485,54 +503,23 @@ export class InstitutionController extends BaseController {
    */
   @AllowAuthorizedParty(AuthorizedParties.aest)
   @Groups(UserGroups.AESTUser)
+  @ApiOkResponse({
+    description: "Institution locations found.",
+  })
+  @ApiNotFoundResponse({
+    description: "Institution locations not found.",
+  })
   @Get("/:institutionId/location-summary")
   async getAllInstitutionLocationSummaryForAEST(
     @Param("institutionId") institutionId: number,
   ): Promise<InstitutionLocationsDetailsDto[]> {
     // get all institution locations.
-    const InstitutionLocationData =
-      await this.locationService.getAllInstitutionLocations(institutionId);
-    return InstitutionLocationData.map(
-      (institutionLocation: InstitutionLocation) => {
-        return {
-          id: institutionLocation.id,
-          name: institutionLocation.name,
-          data: {
-            address: {
-              addressLine1: institutionLocation.data.address?.addressLine1,
-              addressLine2: institutionLocation.data.address?.addressLine2,
-              province: institutionLocation.data.address?.province,
-              country: institutionLocation.data.address?.country,
-              city: institutionLocation.data.address?.city,
-              postalCode: institutionLocation.data.address?.postalCode,
-            },
-          },
-          primaryContact: {
-            primaryContactFirstName:
-              institutionLocation.primaryContact.firstName,
-            primaryContactLastName: institutionLocation.primaryContact.lastName,
-            primaryContactEmail: institutionLocation.primaryContact.email,
-            primaryContactPhone: institutionLocation.primaryContact.phoneNumber,
-          },
-          institution: {
-            institutionPrimaryContact: {
-              primaryContactEmail:
-                institutionLocation.institution.institutionPrimaryContact
-                  .primaryContactEmail,
-              primaryContactFirstName:
-                institutionLocation.institution.institutionPrimaryContact
-                  .primaryContactFirstName,
-              primaryContactLastName:
-                institutionLocation.institution.institutionPrimaryContact
-                  .primaryContactLastName,
-              primaryContactPhone:
-                institutionLocation.institution.institutionPrimaryContact
-                  .primaryContactPhone,
-            },
-          },
-          institutionCode: institutionLocation.institutionCode,
-        } as InstitutionLocationsDetailsDto;
-      },
+    const DesignationAgreementLocationData =
```

Comment:
Local variables should start with a lower case, please see the wiki
https://github.com/bcgov/SIMS/wiki/Development-Standards-and-Decisions#sample-class-example